doc/source/building/working_with_images/custom_fstab_extension.rst

```diff
@@ -0,0 +1,63 @@
+.. _custom_fstab_extension:
+
+Add or Update the Fstab File
+============================
+
+.. sidebar:: Abstract
+
+   This page provides further information for customizing
+   the `/etc/fstab` file which controls the mounting of
+   filesystems at boot time.
+
+In KIWI, all major filesystems that were created at image
+build time are handled by KIWI itself and setup in `etc/fstab`.
+Thus there is usually no need to add entries or change the
+ones added by KIWI. However depending on where the image is
+deployed later it might be required to pre-populate fstab
+entries that are unknown at the time the image is build.
+
+Possible use cases are for example:
+
+* Adding NFS locations that should be mounted at boot time.
+  The information about the NFS location would make this
+  image specific to the target network it later runs in.
+
+* Adding or Changing entries in a read-only root system
+  which becomes effective on first boot but can't be added
+  at that time because of the read-only characteristics.
+
+.. note::
+
+   Modifications to the fstab file are a critical change. If
+   done wrong the risk that the system will not boot exists.
```

Comment:
wrong -> wrongly and the word order is incorrect in this sentence (exist should be at the 2nd position in this sentence)